src/js/translator-mode.js

```diff
@@ -286,4 +286,4 @@ function getCaretCharOffset(element) {
   return caretOffset;
 }
 
-const langTag = () => autoPrefixTag.get().replace(/\$filterLang/gi, languageNameCode[language.get()].code);
+const langTag = () => autoPrefixTag.get().replace(/\$filterLang/gi, languageNameCode[prefixTagReplacementLanguage].code);
```

Comment:
`prefixTagReplacementLanguage` is a `SyncStore`, you need to call `.get()` on it